src/services/events/events-router.ts

```diff
@@ -0,0 +1,22 @@
+import { Router } from "express";
+import { Database } from "../../database";
+import { StatusCodes } from "http-status-codes";
+import { createId } from "@paralleldrive/cuid2";
+import { Event, EventInfo } from "./events-schema";
+
+const authRouter = Router();
+// authRouter.get("/", async (req, res) => {
+// });
+
+authRouter.post("/events", async (req, res, next) => {
```

Comment:
this adds the route onto /event (because that's where the router is being mounted). imo, just do something like / instead, so that you can POST to /api/event/

```suggestion
authRouter.post("/", async (req, res, next) => {
```